src/knex-tables.d.ts

```diff
@@ -176,6 +176,7 @@ declare module "knex/types/tables" {
     onerep_profile_id: number;
     onerep_scan_id: number;
     onerep_scan_reason: Scan["reason"];
+    onerep_scan_status: Scan["status"];
```

Comment:
This might be a premature copy from https://github.com/mozilla/blurts-server/pull/3436?

(Though I suppose Git will automatically resolve that, maybe.)

Reply:
Yes, that’s already in anticipation of the changes in #3436 — otherwise the linter will complain [here](https://github.com/mozilla/blurts-server/pull/3431/files#diff-d701455e388547a86562525e090b5bd1be6d2f06cee5aa5187d602f12253640dR72).